service/src/main/java/de/starwit/service/geojson/GeoJsonMapper.java

```diff
@@ -43,12 +46,19 @@ public static GeoJsonObject mapAreaOccupancies(List<AreaOccupancyObservation> ar
     public static GeoJsonObject mapLineCrossing(LineCrossingObservation lineCrossingObservation) {
         Feature feature = new Feature();
         
-        List<PointEntity> points = lineCrossingObservation.jobEntity().getGeometryPoints();
-
-        feature.setGeometry(new LineString(
-            toLngLatAlt(points.get(0)),
-            toLngLatAlt(points.get(1))
-        ));
+        if (lineCrossingObservation.jobEntity().getGeoReferenced() == true) {
+            List<PointEntity> points = lineCrossingObservation.jobEntity().getGeometryPoints();
+            feature.setGeometry(new LineString(
+                toLngLatAlt(points.get(0)),
```

Comment:
was bedeutet Alt? Vielleicht ausschreiben

Reply:
`Alt` ist Altitude.. `LngLatAlt` heißt der Typ in der GeoJSON-Lib, daher fand ich das halbwegs passend.